src/mainwindow.h

```diff
@@ -53,6 +53,7 @@ class MainWindow : public KXmlGuiWindow
     void setupActions();
     void openMangaFolder();
     void openMangaArchive();
+    void openCloseArchive(bool next);
```

Comment:
I don't really like the name `openCloseArchive`, but I can't really think of a good alternative. `openNextArchive` would be too ambiguous (next as in the next one in the list, or next as in close to?)

Reply:
Why didn't you add 2 functions openNextArchive and openPreviousArchive? 

You don't have to change it now, but I might do it later.

Reply:
Having two functions with the same function body (minus an offset) feels a bit overkill